### DISCUSSION TITLE
CIP 39 - Discussion for mnemonic phrases with error correcting codes

### DISCUSSION THREAD

**Discussion by nategraf** [celo-org/celo-proposals#340] [CIP Discussions]
Discussion issue for "mnemonic phrases with error correcting codes"
https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0039.md

**Comment by jmrossy** [comment]
As you call out in the spec, the CIP-39 phrases are still valid Bip-39 phrases, and Cip-39 wallets should still accept Bip-39 phrases. So this seems like a strictly better option assuming we're confident Cip-39 is still sufficiently strong cryptographically.
